jetty-http/src/main/java/org/eclipse/jetty/http/UriCompliance.java

```diff
@@ -159,7 +159,7 @@ public String getDescription()
 
     public static UriCompliance valueOf(String name)
     {
-        for (UriCompliance compliance : Arrays.asList(DEFAULT, LEGACY, RFC3986, STRICT, SAFE))
+        for (UriCompliance compliance : Arrays.asList(DEFAULT, LEGACY, RFC3986, RFC3986_UNAMBIGUOUS, UNSAFE, SAFE, STRICT))
```

Comment:
Other `*Compliance` classes have a static `KNOWN_MODES` list, and I think for consistency it should be introduced in this class too.

Nit: try to use `List.of()` instead of `Arrays.asList()`.